nautobot/extras/models/jobs.py

```diff
@@ -247,90 +220,28 @@ class Job(PrimaryModel):
     class Meta:
         managed = True
         ordering = ["grouping", "name"]
-        unique_together = ["source", "git_repository", "module_name", "job_class_name"]
+        unique_together = ["module_name", "job_class_name"]
```

Comment:
Do we want a unique_together for grouping/name so you don't end up with duplicates in the UI?

Reply:
We changed `name` to `unique=True` upstream, so there's no longer a need for a `[grouping, name]` unique_together (we had that in 1.x)

Reply:
Didn't we talk about relaxing that and switching to name/grouping instead?

Reply:
We've talked about it yes. :-) That would be returning to the 1.x behavior.

Reply:
Tracked in #2908.